core/arch/arm/plat-ls/main.c

```diff
@@ -1,4 +1,3 @@
-// SPDX-License-Identifier: BSD-2-Clause
```

Comment:
Do not remove the SPDX license identifier please

Reply:
It was never the intention to remove any license related line.

I was working on the top of the release OPTEE Rel 2.6 and created the patch on top of it. 
Since this line was missing from this file in rel 2.6. it got missed.

Reply:
OK, makes sense. So you just need to rebase your branch on top of current master.